cmd/kubelet/app/options/options.go

```diff
@@ -232,6 +233,7 @@ func (s *KubeletServer) AddFlags(fs *pflag.FlagSet) {
 	fs.MarkDeprecated("system-container", "Use --system-cgroups instead. Will be removed in a future version.")
 	fs.StringVar(&s.SystemCgroups, "system-cgroups", s.SystemCgroups, "Optional absolute name of cgroups in which to place all non-kernel processes that are not already inside a cgroup under `/`. Empty for no container. Rolling back the flag requires a reboot. (Default: \"\").")
 
+	fs.BoolVar(&s.CgroupsPerQOS, "cgroups-per-qos", s.CgroupsPerQOS, "Whether the Kubelet should have the QoS cgroup hierarchy, if true top level QoS and pod cgroups are created. Default: false")
```

Comment:
Nit: remove the default: false piece

The default is rendered in help based on s.CgroupsPerQOS already.